.github/workflows/ci.yml

```diff
@@ -85,33 +101,21 @@ jobs:
           sudo apt-get update
           sudo apt install -y xvfb libegl1-mesa libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers
 
-      - name: cargo clippy (no default features)
-        run: cargo clippy --workspace --lib --bins --no-default-features -- -D warnings
-
-      - name: cargo clippy (no default features) (auxiliary)
-        run: cargo clippy --workspace --tests --benches --examples --no-default-features -- -D warnings
-
-      - name: cargo clippy (default features)
-        run: cargo clippy --workspace --lib --bins -- -D warnings
-
-      - name: cargo clippy (default features) (auxiliary)
-        run: cargo clippy --workspace --tests --benches --examples -- -D warnings
-
-      - name: cargo clippy (all features)
-        run: cargo clippy --workspace --lib --bins --all-features -- -D warnings
-
-      - name: cargo clippy (all features) (auxiliary)
-        run: cargo clippy --workspace --tests --benches --examples --all-features -- -D warnings
+      - name: cargo clippy
+        run: cargo hack clippy --workspace --each-feature --optional-deps -- -D warnings
+  
+      - name: cargo clippy (auxiliary)
+        run: cargo hack clippy --workspace --each-feature --optional-deps --tests --benches --examples -- -D warnings
 
       # At the time of writing, we don't have any tests. Nevertheless, it's better to still run this
```

Comment:
This is no longer true :)

Although I suspect our tests could easily get flaky if we add more, given that `wgpu` intentionally serialises their tests. We probably should make something related to their system